### DISCUSSION TITLE
Usage in LAN

### DISCUSSION THREAD

**Discussion by v4mp3rbl4ck** [Syslifters/sysreptor#161] [General]
Currently I deployed the docker in a vmware, to optimize resources I want to use it from the LAN network, but I can only see the application from the localhost, but not from the IPv4 of my machine.

This is how my Iptables looked since I installed docker, any ideas on how to share access to my LAN network?

![rules ipv4](https://github.com/Syslifters/sysreptor/assets/49880796/4f3d9c46-7161-45cf-921c-aa4080c29268)

Configuration of my local network,

Lan 192.168.0.1/23 Interface ens160

**Comment by aronmolnar** [comment]
We recommend to use a webserver for accessing the application via the network.

https://docs.sysreptor.com/setup/webserver/
